.golangci.yml

```diff
@@ -27,6 +26,9 @@ linters-settings:
   depguard:
     list-type: blacklist
     packages:
+      # logging is allowed only by logutils.Log, logrus
```

Comment:
what's this for? It doesn't seem to actually do anything?

Reply:
purged